test/e2e/framework.go

```diff
@@ -1186,19 +1187,19 @@ func (data *TestData) deployFlowAggregator(
 	if err = data.mutateFlowAggregatorConfigMap(ipfixCollector, o); err != nil {
 		return err
 	}
+
+	if o.numFlowAggregatorReplicas > 0 {
```

Comment:
should it be `if o.numFlowAggregatorReplicas > 1`?

Reply:
By allowing `> 0` we can allow scale back down to 1 (with change to secret creation above). I can change to `> 1` though since the other use case isn't really considered right now